src/main/groovy/life/qbic/datamodel/accounting/ProductItem.groovy

```diff
@@ -1,6 +1,7 @@
 package life.qbic.datamodel.accounting
 
 import groovy.transform.EqualsAndHashCode
+import life.qbic.datamodel.dtos.business.services.Product
 
 /**
  * An item with associated costs and quantity
```

Comment:
```suggestion
 * A ProductItem with associated costs and quantity
```

Reply:
The rest of the description needs to be adapted to, replace Cost items with ProductItem in line 9 e.g. and remove the @author tag
(I can't make a suggestion since it's part of the changed code :( )

Reply:
I removed the cost item, but I think we can keep item since it is a short form of ProductItem